racingcar-java/src/test/java/step1/SetTest.java

```diff
@@ -0,0 +1,47 @@
+package step1;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set 크기 확인")
+    void sizeTest() {
+        int setSize = numbers.size();
+        assertThat(setSize).isEqualTo(3);
```

Comment:
```suggestion
        int setSize = numbers.size();
        
        assertThat(setSize).isEqualTo(3);
```
개인 취향이지만, 띄워쓰기를 같이 해주면 가독성이 더 좋아질 것 같아요